core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -244,23 +244,33 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
             }),
             switchMap((payload) =>
                 this.dotPageApiService.save(payload).pipe(
-                    switchMap(() => this.syncEditorData(payload.params)),
-                    tapResponse(
-                        (pageData: DotPageApiResponse) => {
-                            this.patchState((state) => ({
-                                ...state,
-                                editor: pageData,
-                                editorState: EDITOR_STATE.LOADED
-                            }));
-
-                            payload.whenSaved?.();
-                        },
-                        (e) => {
-                            console.error(e);
-                            payload.whenSaved?.();
-                            this.updateEditorState(EDITOR_STATE.ERROR);
-                        }
-                    )
+                    switchMap(() =>
+                        this.syncEditorData(payload.params).pipe(
```

Comment:
```suggestion
                        this.dotPageApiService.get(payload.params).pipe(
```

`syncEditorData` is actually missleading at this point.